xml_analyzer/calculator.py

```diff
@@ -1,63 +1,25 @@
-import xml.etree.ElementTree as Et
-from enum import Enum
-from .formula import Formula
 from .data_container import DataContainer
-
-
-class EquationTypes(Enum):
-    equation = 'equation'
-    inequation_gr_th = 'inequation_gr_th'
-    inequation_sm_th = 'inequation_sm_th'
-    inequation_gr_eq = 'inequation_gr_eq'
-    inequation_sm_eq = 'inequation_sm_eq'
+from .formula import Formula
 
 
 class Calculator:
-    def __init__(self, constraint_property: Et.Element, property_value_mapping: DataContainer):
-        self.constraint_property = constraint_property
-        self.property_value_mapping = property_value_mapping
+    def __init__(self, data: DataContainer):
+        self.data = data
```

Comment:
why is data public

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/d69fb3af4e8f294edfd506a5df7eb7ff6b81aca4